QueueManagerTest/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="3.4.1" targetFramework="net461" />
+  <package id="NUnitTestAdapter.WithFramework" version="2.0.0" targetFramework="net461" />
```

Comment:
I was hoping to keep this at version 3. Can you make it work without regressing the previous version of nunit?